src/linux/backend-v4l2.h

```diff
@@ -285,13 +285,42 @@ namespace librealsense
             std::queue<sync_buffer> _md_queue;
         };
 
+        struct v4l2_device_info
```

Comment:
Where is this struct being used?

Reply:
If it is not used, I will remove it (maybe remained from previous work)

Reply:
Removed